database/insertMockData.js

```diff
@@ -0,0 +1,67 @@
+const randomGenerator = require('./randomGenerator');
+const db = require('./index');
+
+
+const numPics = 348;
+const templateUrl = 'https://s3-us-west-1.amazonaws.com/nappbnbreviews/portait**.jpeg';
+
+const numUsers = randomGenerator.generateInteger(1000, 100);
+const userObjs = [];
+
+for (let i = 0; i < numUsers; i++) {
+  userObjs[i] = {
+    userName: randomGenerator.generateString(null, 10, 3),
+    icon: templateUrl.replace('**', randomGenerator.generateInteger(numPics, 1).toString()),
+  };
+}
+
+let numReviews;
+const reviewObjs = [];
+const zeroReviewRates = {
+  accuracy: 0,
+  communication: 0,
+  cleanliness: 0,
+  location: 0,
+  checkIn: 0,
+  value: 0,
+};
+let avgReviewRates;
+const reviewRates = {};
+
+const addRates = (j, key) => {
+  reviewRates[key] = randomGenerator.generateInteger(5, 3);
+  avgReviewRates[key] = ((avgReviewRates[key] * j) + reviewRates[key]) / (j + 1);
+};
+
+const numRooms = 100;
+const roomObjs = [];
+
+
+for (let i = 0; i < numRooms; i++) {
```

Comment:
According to the style guide looks like is better practice to use higher order functions like forEach, map...  to iterate over arrays.

Reply:
In this case, I don't have an array yet.....